From 2ab9d4d172ac02b4c17bda27db847d0e082f1cc8 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Mon, 14 Sep 2026 12:49:14 +0000
Subject: [PATCH] Refresh the guest package lock

ca-certificates-mozilla 3.128-1 -> 3.129-1
chromium 152.0.7977.82-1 -> 153.0.8010.36-1
dua-cli 2.44.0-1 -> 2.45.0-1
fuse-common 3.18.2-1 -> 3.18.3-1
fuse3 3.18.2-1 -> 3.18.3-1
fzf 0.74.3-1 -> 0.74.4-1
gcr-4 4.4.0.1-1 -> 4.4.1-1
gpu-screen-recorder 6.1.0-1 -> 6.1.2-1
gtk-update-icon-cache 1:4.22.4-1 -> 1:4.22.5-1
gtk4 1:4.22.4-1 -> 1:4.22.5-1
gum 2.0.0-1 -> 2.0.1-1
gvfs 1.60.2-4 -> 1.60.3-1
hyprland 0.56.2-2 -> 0.56.2-3
libcap-ng 0.9.5-1 -> 0.9.6-1
libgcrypt 1.12.3-1 -> 1.12.4-1
libldap 2.6.13-1 -> 2.7.1-1
libportal 0.10.0-1 -> 0.11.0-1
libportal-gtk4 0.10.0-1 -> 0.11.0-1
mkinitcpio 41.1-1 -> 42-1
nss 3.128-1 -> 3.129-1
orc 0.4.43-1 -> 0.4.44-1
systemd 261.2-1 -> 261.3-1
systemd-libs 261.2-1 -> 261.3-1
systemd-resolvconf 261.2-1 -> 261.3-1
systemd-sysvcompat 261.2-1 -> 261.3-1
upower 1.91.3-1 -> 1.91.4-1
xz 5.8.3-1 -> 5.8.4-1
yyjson 0.12.0-1 -> 0.13.0-1
numactl 2.0.19-1 (new)
hyprwayland-scanner (removed)
---
 guest/packages.lock.json | 58 ++++++++++++++++++++--------------------
 1 file changed, 29 insertions(+), 29 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 1173597..add664a 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -35,11 +35,11 @@
     "bubblewrap": "0.12.0-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.128-1",
+    "ca-certificates-mozilla": "3.129-1",
     "ca-certificates-utils": "20240618-1",
     "cairo": "1.18.4-1",
     "cfitsio": "1:4.7.0-1",
-    "chromium": "152.0.7977.82-1",
+    "chromium": "153.0.8010.36-1",
     "clang": "22.1.8-1",
     "compiler-rt": "22.1.8-1",
     "coreutils": "9.11-2",
@@ -60,7 +60,7 @@
     "device-mapper": "2.03.42-1",
     "diffutils": "3.12-2",
     "double-conversion": "3.4.0-1",
-    "dua-cli": "2.44.0-1",
+    "dua-cli": "2.45.0-1",
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
@@ -87,15 +87,15 @@
     "foot": "1.28.0-2",
     "freetype2": "2.14.3-1",
     "fribidi": "1.0.16-2",
-    "fuse-common": "3.18.2-1",
-    "fuse3": "3.18.2-1",
-    "fzf": "0.74.3-1",
+    "fuse-common": "3.18.3-1",
+    "fuse3": "3.18.3-1",
+    "fzf": "0.74.4-1",
     "gawk": "5.4.1-1",
     "gc": "8.2.12-1",
     "gcc": "16.2.1+r23+gd564253eb6c8-1",
     "gcc-libs": "16.2.1+r23+gd564253eb6c8-1",
     "gcr": "3.41.2-2",
-    "gcr-4": "4.4.0.1-1",
+    "gcr-4": "4.4.1-1",
     "gdb": "17.2-1",
     "gdb-common": "17.2-1",
     "gdbm": "1.26-2",
@@ -125,7 +125,7 @@
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-6",
-    "gpu-screen-recorder": "6.1.0-1",
+    "gpu-screen-recorder": "6.1.2-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
     "grep": "3.12-2",
@@ -139,16 +139,16 @@
     "gst-plugins-base-libs": "1.28.7-1",
     "gstreamer": "1.28.7-1",
     "gtest": "1.18.0-1",
-    "gtk-update-icon-cache": "1:4.22.4-1",
+    "gtk-update-icon-cache": "1:4.22.5-1",
     "gtk3": "1:3.24.52-1",
-    "gtk4": "1:4.22.4-1",
+    "gtk4": "1:4.22.5-1",
     "gtk4-layer-shell": "1.3.0-1",
     "guile": "3.0.11-1",
-    "gum": "2.0.0-1",
+    "gum": "2.0.1-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
-    "gvfs": "1.60.2-4",
+    "gvfs": "1.60.3-1",
     "gzip": "1.14-2",
     "harfbuzz": "14.4.0-1",
     "hdf5": "2.2.0-1",
@@ -161,7 +161,7 @@
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
-    "hyprland": "0.56.2-2",
+    "hyprland": "0.56.2-3",
     "hyprland-guiutils": "0.2.2-3",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
@@ -169,7 +169,6 @@
     "hyprsunset": "0.4.0-3",
     "hyprtoolkit": "0.5.4-6",
     "hyprutils": "0.14.2-1",
-    "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
@@ -226,7 +225,7 @@
     "libbsd": "0.12.2-2",
     "libbytesize": "2.12-3",
     "libcap": "2.78-1",
-    "libcap-ng": "0.9.5-1",
+    "libcap-ng": "0.9.6-1",
     "libcdio": "2.4.0-1",
     "libcdio-paranoia": "10.2+2.0.2-3",
     "libcgif": "0.5.4-1",
@@ -259,7 +258,7 @@
     "libfreeaptx": "0.2.2-1",
     "libfyaml": "0.9.6-3",
     "libgcc": "16.2.1+r23+gd564253eb6c8-1",
-    "libgcrypt": "1.12.3-1",
+    "libgcrypt": "1.12.4-1",
     "libgfortran": "16.2.1+r23+gd564253eb6c8-1",
     "libgirepository": "1.86.0-2",
     "libgit2": "1:1.9.7-1",
@@ -286,7 +285,7 @@
     "libksba": "1.8.1-1",
     "liblc3": "1.1.3-2",
     "libldac": "2.0.2.6-1",
-    "libldap": "2.6.13-1",
+    "libldap": "2.7.1-1",
     "liblqr": "0.4.3-1",
     "liblsan": "16.2.1+r23+gd564253eb6c8-1",
     "libluv": "1.52.1-1",
@@ -328,8 +327,8 @@
     "libplacebo": "7.360.1-6",
     "libplist": "2.7.0-3",
     "libpng": "1.6.58-2",
-    "libportal": "0.10.0-1",
-    "libportal-gtk4": "0.10.0-1",
+    "libportal": "0.11.0-1",
+    "libportal-gtk4": "0.11.0-1",
     "libproxy": "0.5.12-1",
     "libpsl": "0.21.5-2",
     "libpulse": "17.0+r98+gb096704c0-1",
@@ -439,7 +438,7 @@
     "mdadm": "4.6-2",
     "mesa": "1:26.2.2-1",
     "minizip": "1:1.3.2-3",
-    "mkinitcpio": "41.1-1",
+    "mkinitcpio": "42-1",
     "mkinitcpio-busybox": "1.36.1-1",
     "mobile-broadband-provider-info": "20251101-1",
     "mpdecimal": "4.0.1-3",
@@ -461,7 +460,8 @@
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.128-1",
+    "nss": "3.129-1",
+    "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
     "omacut": "0.4.0-1",
@@ -479,7 +479,7 @@
     "openssh": "10.5p1-1",
     "openssl": "3.6.4-1",
     "opus": "1.6.1-1",
-    "orc": "0.4.43-1",
+    "orc": "0.4.44-1",
     "osinfo-db": "20260812-1",
     "p11-kit": "0.26.5-1",
     "pacman": "7.1.0.r9.g54d9411-2",
@@ -563,10 +563,10 @@
     "starship": "1.26.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "261.2-1",
-    "systemd-libs": "261.2-1",
-    "systemd-resolvconf": "261.2-1",
-    "systemd-sysvcompat": "261.2-1",
+    "systemd": "261.3-1",
+    "systemd-libs": "261.3-1",
+    "systemd-resolvconf": "261.3-1",
+    "systemd-sysvcompat": "261.3-1",
     "tar": "1.35-5",
     "tensaku": "0.29.0-1",
     "tesseract": "5.5.3-1",
@@ -595,7 +595,7 @@
     "udisks2": "2.11.2-1",
     "unibilium": "2.1.4-1",
     "unzip": "6.0-23",
-    "upower": "1.91.3-1",
+    "upower": "1.91.4-1",
     "util-linux": "2.42.3-1",
     "util-linux-libs": "2.42.3-1",
     "uwsm": "0.26.7-1",
@@ -648,10 +648,10 @@
     "xorgproto": "2025.1-1",
     "xvidcore": "1.3.7-4",
     "xxhash": "0.8.3-1",
-    "xz": "5.8.3-1",
+    "xz": "5.8.4-1",
     "yay": "13.0.1-1",
     "yoga": "3.2.1-1",
-    "yyjson": "0.12.0-1",
+    "yyjson": "0.13.0-1",
     "zbar": "0.23.93-7",
     "zeromq": "4.3.5-3",
     "zimg": "3.0.6-2",
-- 
2.55.0

